### ISSUE TITLE
ViewDestroyedError: Attempt to use a destroyed view: detectChanges

### ISSUE THREAD

**Issue by surjit** [flauc/angular2-notifications#314]
```
zone.js:192 Uncaught Error: ViewDestroyedError: Attempt to use a destroyed view: detectChanges
    at viewDestroyedError (core.js:9800)
    at Object.debugUpdateDirectives [as updateDirectives] (core.js:14652)
    at checkAndUpdateView (core.js:13802)
    at callWithDebugContext (core.js:15056)
    at Object.debugCheckAndUpdateView [as checkAndUpdateView] (core.js:14593)
    at ViewRef_.detectChanges (core.js:11577)
    at eval (angular2-notifications.umd.js:548)
    at ZoneDelegate.invoke (zone.js:388)
    at Object.onInvoke (core.js:4749)
    at ZoneDelegate.invoke (zone.js:387)
```

**Comment by davidalee** [comment]
I'm seeing this issue as well. Not doing anything crazy, I have the same setup as described in the documentation.

Not sure how to reproduce... it seems to occur randomly.

**Comment by davidalee** [comment]
There seems to be an issue created for this already here: https://github.com/flauc/angular2-notifications/issues/296

**Comment by flauc** [comment]
Closing in favor of #296

**Comment by AbhijeetThakur** [comment]
I'm facing this issue in version 0.9.6. Is this issue fixed in **new version 1.0.2**?

**Comment by flauc** [comment]
No, actually it isn't, but it was addressed in #328 I will be merging that later today.

**Comment by sapariyaravi** [comment]
Is this issue fixed? I am facing the same.
